src/pages/Popup/App.tsx

```diff
@@ -21,14 +28,70 @@ function App() {
     ],
   };
 
+  function incrementNavigation(e: React.MouseEvent) {
+    const target = e.target as HTMLButtonElement;
+    updateNavigation(target.value as Pages);
+  }
+
   return (
     <div className="App">
       <header className="App-header">
-        <LogIn />
-        {/* <TaskHistory />
-        <TaskDisplay /> */}
-        {/* <TaskTracker /> */}
-        {/* <TaskEditor TaskData={dummyTask} /> */}
+        <p>current navigation: {current_navigation_state}</p>
+        <div className="nav-container">
+          <button
+            className="nav-button"
+            value={'Login'}
+            onClick={incrementNavigation}
+          >
+            Login
+          </button>
+          <button
+            className="nav-button"
+            value={'TaskDisplay'}
+            onClick={incrementNavigation}
+          >
+            Task Display
+          </button>
+          <button
+            className="nav-button"
+            value={'TaskEdit'}
+            onClick={incrementNavigation}
+          >
+            Task Edit
+          </button>
+          <button
+            className="nav-button"
+            value={'TaskAdd'}
+            onClick={incrementNavigation}
+          >
+            Task Add
+          </button>
+          <button
+            className="nav-button"
+            value={'TaskTimer'}
+            onClick={incrementNavigation}
+          >
+            Task Timer
+          </button>
+          <button
+            className="nav-button"
+            onClick={() => revertToPreviousState()}
+          >
+            back
+          </button>
+        </div>
+        <TaskHistory />
+        {current_navigation_state === 'Login' ? (
```

Comment:
I would use if statements not ternary

Reply:
I would have used a switch statement instead not going to lie. I'd prefer that over nesting the many if and else ifs